ext/curl/tests/curl_setopt_CURLOPT_FOLLOWLOCATION.phpt

```diff
@@ -0,0 +1,31 @@
+--TEST--
+CURLOPT_FOLLOWLOCATION values
+--EXTENSIONS--
+curl
+--SKIPIF--
+<?php
+include 'skipif-nocaddy.inc';
+if (curl_version()['version_number'] < 0x080d00) die('skip requires curl >= 8.13.0');
+?>
+--FILE--
+<?php
+
+for ($i = 0; $i <= CURLFOLLOW_FIRSTONLY; $i ++) {
```

Comment:
You're making assumptions here about the specific values that these constants have. Unlikely to change, but not a fan to depend on this.
I propose to use a foreach loop with an array containing the values you want to test.